dev_guide/routes.adoc

```diff
@@ -133,10 +134,33 @@ custom certificates. Any non-SNI traffic received on port 443 is handled with
 TLS termination and a default certificate, which may not match the requested
 host name, resulting in validation errors.
 ====
+endif::[]
 
 Further information on all types of
 xref:../architecture/core_concepts/routes.adoc#secured-routes[TLS termination]
 as well as
 xref:../architecture/core_concepts/routes.adoc#path-based-routes[path-based
 routing] are available in the
 xref:../architecture/core_concepts/routes.adoc#architecture-core-concepts-routes[Architecture section].
+
+ifdef::openshift-online[]
+[[custom-route-and-hosts-and-certificates-restrictions]]
+== Restrictions
+
+Currently, custom route hosts are not permitted. Instead, the following host
+template is enforced on all user routes:
+
+----
+<route-name>-<namespace>.44fs.preview.openshiftapps.com
+----
+
+Also, custom certificates are not permitted. Only unencrypted routes, edge
+routes using the default certificate, and passthrough routes work. Edge routes
+with custom certificates and re-encrypt routes (which necessarily have custom
+certificates) will not work.
+
+These restrictions are enforced in the router. Inspecting the
+route in the console or using `oc get routes` will display the host and
+certificates specified by the user. However, any custom host will not be
+respected, and a route with custom certificates will not work.
+endif::[]
```

Comment:
@ahardin-rh A lot of "will"-ness. Can we change the tense to non-future?

"custom hosts are not respected"
"routes with custom certificates do not work"